packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2283,11 +2283,18 @@ This operation can only be performed on users that do not have a `DEPROVISIONED`
 | userId    | ID of user                                                                            | URL        | String   | TRUE     |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    |
 
+> Note: You can also perform user deactivation asynchronously.
```

Comment:
the word Note and the colon are bold. I don't think that came across in my comments. Sorry. So > and then two asterisks Note: and two asterisks. Sorry about that.

Reply:
Updated.